### DISCUSSION TITLE
Qiitaにサロン的なスペースを導入してはどうか、と。

### DISCUSSION THREAD

**Discussion by fygar256** [increments/qiita-discussions#906] [要望・アイデア・不具合]
例えば、Qiitaにサロン的なスペースがあると、メンバー同士の交流が持てて楽しくなるんではないかと思います。
反面、今、Qiitaがこのクォリティを維持しているのは、あえてSNS的なものを避けて技術記事に絞っていることでではないかと
思われます。僕には原因が判らないので、提案させていただきました。
反対意見、賛成意見があると思いますが、一度、Qiitaのクォリティが崩れると元に戻りそうにないので、慎重にこの件はご検討
願いたいと思います。今のままでうまく行ってるので、これでもいいかと思います。それでは、よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@fygar256 
ご提案ありがとうございます！

Qiita上にサロン的な機能を設計するかどうかは、まだ具体的に検討したことがないのですが、
Qiita上の機能に限らず、どんな事が実現できるといいのか、みなさんからのご意見も参考にしたいと思います。
（現状ですと、一番近い機能は質問機能の「意見交換」かと思いますが、少し違いますよね…？）

また今後、メンバー同士の交流を持てるようなオフラインイベント等を予定しているため
イベントの反応やみなさんのご意見をお聞きしながら、検討したいと思います！✨

**Comment by fygar256** [comment]
お返事どうもありがとうございます。
close
